config/locales/en/admin.yml

```diff
@@ -278,6 +278,19 @@ en:
         Default Country:
           name: Default Country
           description: Select the default country for new family records.
+        Map Zoom Level:
+          name: Map Zoom Level
+          description: Select the zoom level to use on the maps displayed on the site.
+        Zoom Level 12:
+          name: "Zoomed out"
+        Zoom Level 13:
+          name: "Normal"
+        Zoom Level 14:
+          name: "Zoomed In"
+        Zoom Level 15:
+          name: "Zoomed in closer"
+        Zoom Level 16:
+          name: "Super zoomed in"
```

Comment:
On second thought, let's just leave these strings out of the locale (will simplify translation efforts) and use `(12..16).to_a` for the options array.